loadtest/store/memstore/random.go

```diff
@@ -6,7 +6,6 @@ package memstore
 import (
 	"errors"
 	"math/rand"
-	"reflect"
```

Comment:
This is what makes this PR totally worth it in my opinion.

Reply:
@isacikgoz will be sad :joy: